shlr/spp/spp.h

```diff
@@ -83,15 +85,8 @@ struct Tag *tags = (struct Tag *)&x##_tags; \
 struct Arg *args = (struct Arg *)&x##_args; \
 struct Proc *proc = &x##_proc;
 
-typedef struct s_strbuf_t {
-	int len;
-	char *ptr;
-	int ptrlen;
-	char buf[64];
-} SStrBuf;
-
 typedef struct {
-	SStrBuf *cout;
+	RStrBuf *cout;
```

Comment:
you can't use RStrBuf here as it does not exist in spp.... Drop these changes please.

Reply:
What I propose is dropping fix attempts for strbuf api/spp connection and just fixing the rest until a r_util project is set then further spp project update separately. What do you think ?

Reply:
I agree. Just leave those warnings for now... Unfortunately.